cclient/README.md

```diff
@@ -60,11 +74,13 @@ cc_binary(
 
 ```
 
-C Client application example [iota_cclient_example](https://github.com/oopsmonk/iota_cclient_example)  
+Second include `cclient/api/core/core_api.h` and `cclient/api/extended/extended_api.h` in the source code.  
+
+This is the example project: [iota_cclient_example](https://github.com/oopsmonk/iota_cclient_example)  
```

Comment:
We should probably host examples in entangled at some point

Reply:
This is an example of bazel project, will not host in entangled, but If you were talking about the code implementation, yes, it will be.